src/wallet/api/wallet.cpp

```diff
@@ -1149,7 +1147,7 @@ PendingTransaction *WalletImpl::createTransaction(const string &dst_addr, const
                 de.is_subaddress = info.is_subaddress;
                 dsts.push_back(de);
                 transaction->m_pending_tx = m_wallet->create_transactions_2(dsts, fake_outs_count, 0 /* unlock_time */,
-                                                                          adjusted_priority,
+                                                                          static_cast<uint32_t>(priority),
```

Comment:
Looks like it's already uint32_t.

Reply:
It's `PendingTransaction::Priority` fwiw.